components/breadcrumbs/default.htm

```diff
@@ -1,11 +1,11 @@
 {% set crumbOptions = __SELF__.options %}
 {% if currentCrumb.show_crumb %}
-<ol class="{{crumbOptions['main-ol-class']}}">
+<ol itemscope itemtype="http://schema.org/BreadcrumbList" class="{{crumbOptions['main-ol-class']}}">
```

Comment:
May be it will be more correctly to use `https` instead `http` ?

Reply:
Hi! You're right! I did not think about it